datafusion/physical-expr/src/expressions/negative.rs

```diff
@@ -236,9 +179,6 @@ mod tests {
         test_array_negative_op!(Int64, 23456i64, 12345i64);
         test_array_negative_op!(Float32, 2345.0f32, 1234.0f32);
         test_array_negative_op!(Float64, 23456.0f64, 12345.0f64);
-        test_array_negative_op_intervals!(YearMonth, 2345i32, 1234i32);
-        test_array_negative_op_intervals!(DayTime, 23456i64, 12345i64);
-        test_array_negative_op_intervals!(MonthDayNano, 234567i128, 123456i128);
```

Comment:
This is moved into sqllogic, unfortunately I couldn't find a way to get a MonthDayNano type, but this is better than nothing

Reply:
I think you can do something like this

```
select arrow_cast(interval '30 minutes', 'Interval(MonthDayNano)');
```

Reply:
Arrow cast doesn't appear to support intervals in my experiments